node/src/benchmarking.rs

```diff
@@ -138,8 +142,9 @@ pub fn create_benchmark_extrinsic(
 		.checked_next_power_of_two()
 		.map(|c| c / 2)
 		.unwrap_or(2) as u64;
+
 	let extra: runtime::SignedExtra = (
-		frame_system::CheckNonZeroSender::<runtime::Runtime>::new(),
+		//frame_system::CheckNonZeroSender::<runtime::Runtime>::new(),
```

Comment:
?

Reply:
I noticed that the `SignedExtra` definitions in `benchmarking.rs` and `runtime`(https://github.com/ggxchain/ggxnode/blob/a6ae9f3b125970e4f2023a03b4f7c85fca075400/runtime/brooklyn/src/lib.rs#L155) are inconsistent.

Specifically, `benchmarking.rs` includes an extra item: `frame_system::CheckNonZeroSender::<runtime::Runtime>::new()`. Due to this inconsistency, compiling with `--features runtime-benchmarks` fails.

I understand that this effectively bypasses a check. If necessary, I can add this item to the `runtime` definition to ensure consistency on both sides.